src/sentry/utils/integrationdocs.py

```diff
@@ -122,9 +123,14 @@ def sync_docs(quiet=False):
 
     dump_doc("_platforms", {"platforms": platform_list})
 
+    executor = multiprocessing.dummy.Pool(min(32, multiprocessing.cpu_count() * 5))
```

Comment:
Right now, there's 16 things in `data["platforms"]`, 16 I/O bound threads is essentially free, so why are you capping it at 32? I'd recommend just having `len(data["platforms"])` threads.

Reply:
See my other comment :)

Reply:
I saw, but your other comment doesn't address this.

Reply:
I think I can add `len(data["platforms"])` inside the `min()` call but I honestly think this is mostly a nit as we are not spawning processes here.

Reply:
Fair.